runtime/src/bank.rs

```diff
@@ -2764,6 +2765,32 @@ impl Bank {
         }
     }
 
+    #[allow(dead_code)]
+    fn hash_rewards_into_partitions(
+        stake_rewards: &StakeRewards,
+        rewarded_epoch: Epoch,
+        rewards: u64,
+        num_partitions: usize,
+        thread_pool: &ThreadPool,
+    ) -> Vec<StakeRewards> {
+        let seed = rewarded_epoch ^ rewards;
+        let hasher = create_epoch_reward_hasher(seed);
+
+        let mut result = vec![vec![]; num_partitions];
+
+        thread_pool.install(|| {
```

Comment:
have we benched this to justify a thread?

Reply:
benchmark 600K
hash: 2.355ms (parallel)
hash: 14.957ms (sequential)

Reply:
in isolation though, right? not with the 500 other threads that we stuff down the scheduler's throat

Reply:
yes. it is.

but this piece of code runs at the epoch boundary block, everything is waiting on the reward to finish. We should use as much resource as available on the machine, to finish the rewards as far as possible.